Configuration_adv.hpp

```diff
@@ -534,6 +534,18 @@
     #endif
 #endif
 
+#if DEW_HEATER == 1
+    // Careful with those values, they might melt your lens.
+    // Make doubly sure your setup can handle 12V.
+    // this is the PWM duty cycle with vales from 0...255
+    #if !defined(DEW_HEATER_1_MAX)
+        #define DEW_HEATER_1_MAX 255
```

Comment:
These values should not default to full power. Maybe something like 10%?

Reply:
I disagree, I think this approach is correct - the max value should be 255, the default operating setting is what should be lower.

Reply:
It's what is currently used and if people have 12v heaters it should be fine. if you have 5v you need to test anyway.